call.go

```diff
@@ -113,6 +133,22 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 			o.after(&c)
 		}
 	}()
+
+	c.traceInfo.tr = trace.New("Sent."+familyForMethod(method), method)
+	defer c.traceInfo.tr.Finish()
+	c.traceInfo.firstLine.client = true
+	if deadline, ok := ctx.Deadline(); ok {
+		c.traceInfo.firstLine.deadline = deadline.Sub(time.Now())
+	}
+	c.traceInfo.tr.LazyLog(&c.traceInfo.firstLine, false)
+	// TODO(dsymonds): Arrange for c.traceInfo.firstLine.remoteAddr to be set.
+	defer func() {
+		if err != nil {
+			c.traceInfo.tr.LazyLog(&lazySprintf{"%v", []interface{}{err}}, true)
+			c.traceInfo.tr.SetError()
+		}
+	}()
+
```

Comment:
Hmm. It looks like the benchmark stuff is a bit broken for me. I'll send you a pull request to fix one small thing, but that only uncovered a crash in it, which I'll file separately.